src/herder/HerderSCPDriver.cpp

```diff
@@ -692,6 +703,9 @@ HerderSCPDriver::combineCandidates(uint64_t slotIndex,
 
     // take the txSet with the biggest size, highest xored hash that we have
     TxSetFramePtr bestTxSet;
+    TimePoint bestTxSetCloseTime = 0;
```

Comment:
Relating to what @marta-lokhova commented below, I don't think you actually need this at all? Or am I missing something?

Reply:
If you mean that I could replace `bestTxSetTime = sv.closeTime` with `comp.closeTime = sv.closeTime` (under `newCloseTimeBehavior`), then yeah, I could do that.  I thought it was clearer to have the decision point based on whether we're using pre- or post-CAP-34 code in one place, namely here:

```
    if (newCloseTimeBehavior)
    {
        comp.closeTime = bestTxSetCloseTime;
        comp.ext.v(STELLAR_VALUE_SIGNED);
        comp.ext.lcValueSignature() = highestTxSetSignature;
    }
    else
    {
        comp.closeTime = maxCloseTime;
        comp.ext.v(STELLAR_VALUE_BASIC);
    }
```

I thought that that spelled out in one place exactly what the difference was between the way that the old and new protocols constructed `comp`, whereas if I didn't use the locals, then the assignment for 'newCloseTimeBehavior' would be up in the loop (and possibly done repeatedly) whereas the one for `!newCloseTimeBehavior` would be after the loop, making it less immediately obvious what the difference was, and perhaps making it unclear why we were not doing a post-loop assignment after all in the `newCloseTime` case.  It appears not to be providing that clarity, so I'll change it.